requirments.txt

```diff
@@ -0,0 +1,15 @@
+Babel==2.3.4
+decorator==4.0.10
+digExtractor>=0.1.8
+esmre==0.3.1
+idna==2.1
+jsonpath-rw==1.4.0
```

Comment:
Can you go through and make sure these packages are explicitly required by this extractor and remove them if they aren't?  Also remove any transitive dependencies like the jsonpath stuff.